.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "usb/usb_host_uvc/libuvc"]
-	path = usb/usb_host_uvc/libuvc
-	url = https://github.com/libuvc/libuvc.git
+[submodule "host/usb_host_uvc/libuvc"]
+	path = host/usb_host_uvc/libuvc
+	url = git@github.com:libuvc/libuvc.git
```

Comment:
This will probably not work for folks who don't have SSH-based authentication set up in Github.

You can use a relative URL, though: `../../libuvc/libuvc.git` — this will follow the same protocol as was used to clone espressif/esp-usb. However given that you likely won't push to libuvc/libuvc, always cloning it over HTTPS should be fine?

Reply:
Thanks! I must have missed it amid so many index changes